packages/elements/src/components/ino-table/readme.md

```diff
@@ -5,21 +5,28 @@ The component is based on the mdc-data-table.
 
 ## Usage
 
-The component is composed of multiple sub-components and can be used as follows:
+The component comprises two sub-components: `ìno-table-row` and `ino-table-cell`. The `ino-table-row` can be used to
+add a row to the table. To add a header row, all you have to do is to add the `ino-header-row` property to the component.
+Correspondingly, the `ino-table-cell` can be used to add a column to the table. You should always set the `ino-numeric` 
+property to true if the column contains numeric values. 
+The following examples illustrate how the `ino-table` can be used:
 
 ### Web Component
 
 ```html
 <ino-table>
-    <ino-table-row slot="header">
+    <ino-table-row slot="header" ino-header-row>
```

Comment:
Hi ✌️ do we still need to add the `slot="header"` attribute since it not mentioned in the usage description text :)

Reply:
True - I'll add it right away. 🙂